modules/seqexec/server/src/main/scala/seqexec/server/tcs/TcsControllerEpics.scala

```diff
@@ -270,8 +272,36 @@ object TcsControllerEpics extends TcsController {
   private val tcsTimeout = Seconds(60)
   private val agTimeout = Seconds(60)
 
-  private def willMove(current: EpicsTcsConfig, demand: TcsConfig): Boolean =
-    demand.tc.offsetA.exists(_ =!= current.instrumentOffset)
+  private val pwfs1OffsetThreshold = Arcseconds(0.01)/FOCAL_PLANE_SCALE
+  private val pwfs2OffsetThreshold = Arcseconds(0.01)/FOCAL_PLANE_SCALE
+  private def aoOffsetThreshold(instrument: Instrument): Option[Length] = instrument match {
+    case Instrument.Nifs  => (Arcseconds(0.01)/FOCAL_PLANE_SCALE).some
+    case Instrument.Niri  => (Arcseconds(3.0)/FOCAL_PLANE_SCALE).some
+    case Instrument.Gnirs => (Arcseconds(3.0)/FOCAL_PLANE_SCALE).some
+    case _                => none
+  }
```

Comment:
The old Seqexec used different thresholds for different instruments when guiding with Altair. I followed that, but I am not sure why it is like this.